backend/config/routes.rb

```diff
@@ -132,7 +132,9 @@
       collection do
         post :update_positions
       end
-      resources :taxons
+      resources :taxons do
+        resource :attachment, controller: '/spree/admin/taxons/attachment', only: [:destroy]
```

Comment:
I think you can just use `taxons/attachment` as the `controller` value.

Reply:
You're right! Thanks! 👍